packages/blade/src/components/Skeleton/_decisions/decisions.md

```diff
@@ -0,0 +1,327 @@
+# Skeleton <!-- omit in toc -->
+Skeleton Loader is a static / animated placeholder for the information that is still loading. It mimic the structure and look of the entire view. 
+
+The skeleton blocks can have 3 types of shape
+- circle 
+- rectangle
+- pill
+
+Each of these shapes can be achieved by adjusting the width, height and border radius of the blocks. Properties of skeleton loaders.
+
+<img src="./skeleton-thumbnail.png" width="380" />
+
+## Design
+- [Figma - Skeleton](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=16508%3A258522&mode=dev)
+
+
+## API
+
+| Prop     | Type        | Default | Description                                  | Required |
+| -------- | ----------- | ------- | -------------------------------------------- | -------- |
+| contrast | `low, high` | `low`   | The contrast of the Skeleton to be rendered. |          |
+
+Skeleton will have subset of Box props to help users layout the Skeleton blocks accordingly. 
+
+Layout: 
+
+- borderRadius
+- height
+- minHeight
+- maxHeight
+- width
+- minWidth
+- maxWidth
+
+Flex:
```

Comment:
why do we need Flex for skeleton?

Reply:
We will need to support all layout related props otherwise consumers won't be able to replicate complex structures like Card, Alert etc. 

![image](https://github.com/razorpay/blade/assets/35374649/2516f7f0-ac41-40f3-b95b-f8081a43ff69)